examples/tokyo_travel_planner.py

```
@@ -0,0 +1,100 @@
+"""
+Direct Tokyo travel planning using LLM without QADI methodology.
+"""
+
+import asyncio
+import os
+from pathlib import Path
+from rich.console import Console
+from rich.panel import Panel
+from rich.markdown import Markdown
+
+from mad_spark_alt.core.llm_provider import LLMProvider, LLMRequest, llm_manager, setup_llm_providers
```

Ruff (0.11.9)

`mad_spark_alt.core.llm_provider.setup_llm_providers` imported but unused

Remove unused import: `mad_spark_alt.core.llm_provider.setup_llm_providers`

(F401)